Tests/UnitTests/OrderTests.swift

```diff
@@ -126,48 +126,108 @@ class OrderTests: XCTestCase {
     }
     
     func testCustomerInitialization_hashingEmail() {
+        // Arrange
         let id = "123"
         let email = "betty@usebutton.com"
         let emailSha256 = "c399e8d0e89e9f09aa14a36392e4cb0d058ab28b16247e80eab78ea5541a20d3"
         let advertisingId = "123"
-        
+
+        // Act
         let customer = Order.Customer()
         customer.id = id
         customer.setEmail(email)
         customer.advertisingId = advertisingId
 
+        // Assert
         XCTAssertEqual(customer.id, id)
         XCTAssertEqual(customer.emailSha256, emailSha256)
         XCTAssertEqual(customer.advertisingId, advertisingId)
     }
     
     func testCustomerInitialization_providingHashedEmail() {
+        // Arrange
         let id = "123"
         let emailSha256 = "1234567"
         let advertisingId = "123"
-        
+
+        // Act
         let customer = Order.Customer()
         customer.id = id
         customer.emailSha256 = emailSha256
         customer.advertisingId = advertisingId
-        
+
+        // Assert
         XCTAssertEqual(customer.id, id)
         XCTAssertEqual(customer.emailSha256, emailSha256)
         XCTAssertEqual(customer.advertisingId, advertisingId)
     }
     
     func testCustomerInitialization_hashingCapitalizedEmail() {
+        // Arrange
         let id = "123"
         let email = "BETTY@usebutton.com"
         let emailSha256 = "c399e8d0e89e9f09aa14a36392e4cb0d058ab28b16247e80eab78ea5541a20d3"
         let advertisingId = "123"
 
+        // Act
         let customer = Order.Customer()
         customer.id = id
         customer.setEmail(email)
         customer.advertisingId = advertisingId
-        
+
+        // Assert
         XCTAssertEqual(customer.emailSha256, emailSha256)
     }
 
+    func testLineItemInitialization_requiredPropertiesOnly() {
+        // Arrange
+        let identifier = "unique-id-123"
+        let total: Int64 = 4000
+
+        // Act
+        let lineItem = Order.LineItem(identifier: identifier, total: total)
+
+        // Assert
+        XCTAssertEqual(lineItem.identifier, identifier)
+        XCTAssertEqual(lineItem.total, total)
+        XCTAssertEqual(lineItem.quantity, 1)
+        XCTAssertNil(lineItem.itemDescription)
+        XCTAssertNil(lineItem.sku)
+        XCTAssertNil(lineItem.upc)
+        XCTAssertNil(lineItem.category)
+        XCTAssertNil(lineItem.attributes)
+    }
+
+    func testLineItemInitialization_allProperties() {
```

Comment:
Are we covering serialization elsewhere?

Reply:
When we have a finalized model, then a test for serialization will be added